### ISSUE TITLE
When Adding a new network, we need to strip the `http://` prefix or show an error

### ISSUE THREAD

**Issue by lucca65** [EOSEssentials/Scatter#115]
To avoid problems like this:

![image](https://user-images.githubusercontent.com/1082127/41740483-92565ada-756e-11e8-943d-158eee70c036.png)

**Comment by nsjames** [comment]
Looking at this now, i'm not sure how you even got those in there.
https://github.com/EOSEssentials/Scatter/blob/master/src/views/NetworkView.vue#L60

It should have stripped the `http://` off when you input it.

**Comment by lucca65** [comment]
Yeah, probably thats because I'm adding the network from javascript using the suggested settings: 

```javascript
const network = {
  blockchain: 'eos',
  host: 'http://dev.cryptolions.io',
  port: 38888
};
```

Probably it doesn't go through Vue's logic

**Comment by nsjames** [comment]
Oh. That explains it :)
I'll fix that with an error message about not using protocols within the network since there's a protocol field on the network itself.
